app/firebase/functions/src/platforms/nanopub/nanopub.service.ts

```diff
@@ -60,9 +62,33 @@ export class NanopubService
     if (!params) {
       throw new Error('Missing params');
     }
+    if (!userId) {
```

Comment:
these changes break the signup tests, which don't pass userId when calling the `getSignupContext` for the Nanopub platform. We need to decide how to handle providing a username and name in the intro nanopub.

Reply:
@pepoospina